### PR TITLE
feat(plasma-web-docs): Extend docs 

### PR DESCRIPTION
**Pull request by Yakutoc**
Компоненты 

- AudioPlayer
- Editable
- ElasticGrid
- PreviewGallery
- Upload
- Slider

были перенесены в **plasma-hope** и стали доступны в **plasma-web**. 

Теперь по этим компонентам так же добавленна соответствующая [документация](https://plasma.sberdevices.ru/web/).     

:baby_chick: Download canary assets:

[colors--canary.379.4342318097.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.379.4342318097.xml)  
[color_metro_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.379.4342318097.swift)  
[color_metro_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.379.4342318097.kt)  
[color_metro_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.379.4342318097.ts)  
[color_sberHealth_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.379.4342318097.swift)  
[color_sberHealth_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.379.4342318097.kt)  
[color_sberHealth_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.379.4342318097.ts)  
[color_sbermarket_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.379.4342318097.swift)  
[color_sbermarket_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.379.4342318097.kt)  
[color_sbermarket_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.379.4342318097.ts)  
[color_sberprime_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.379.4342318097.swift)  
[color_sberprime_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.379.4342318097.kt)  
[color_sberprime_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.379.4342318097.ts)  
[color_selgros_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.379.4342318097.swift)  
[color_selgros_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.379.4342318097.kt)  
[color_selgros_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.379.4342318097.ts)  
[color_smbusiness_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.379.4342318097.swift)  
[color_smbusiness_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.379.4342318097.kt)  
[color_smbusiness_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.379.4342318097.ts)  
[PlasmaTokensColor--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.379.4342318097.swift)  
[shadow_sbermarket_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.379.4342318097.ts)  
[typo_mage_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.379.4342318097.swift)  
[typo_mage_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.379.4342318097.kt)  
[typo_mage_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.379.4342318097.ts)  
[typo_plasma_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.379.4342318097.swift)  
[typo_plasma_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.379.4342318097.kt)  
[typo_plasma_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.379.4342318097.ts)  
[typo_ruler_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.379.4342318097.swift)  
[typo_ruler_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.379.4342318097.kt)  
[typo_ruler_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.379.4342318097.ts)  
[typo_sage_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.379.4342318097.swift)  
[typo_sage_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.379.4342318097.kt)  
[typo_sage_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.379.4342318097.ts)  
[typo_sbermarket_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.379.4342318097.swift)  
[typo_sbermarket_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.379.4342318097.kt)  
[typo_sbermarket_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.379.4342318097.ts)  
[typo_soulmate_ios-swift--canary.379.4342318097.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.379.4342318097.swift)  
[typo_soulmate_kotlin--canary.379.4342318097.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.379.4342318097.kt)  
[typo_soulmate_react-native--canary.379.4342318097.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.379.4342318097.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-379/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-379/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-379/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-379/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-379/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-379/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-379/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-379/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-379/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-379/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-379/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-379/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-379/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-379/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-379/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-379/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-379/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-379/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-379/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-379/temple-storybook/

File: website/plasma-web-docs/docs/components/Slider.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <section>
+            <Slider onChangeCommitted={onChangeCommittedHandle} min={0} max={100} value={value} />
+        </section>
+    );
+}
+```
+
+## Примечание
+
+**Только для приложения Салют.**
```
Comment (Yeti-or) [inline website/plasma-web-docs/docs/components/Slider.mdx:55]: не-а, это для plasma-ui справидливо

File: website/plasma-web-docs/docs/components/Slider.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+export function App() {
+    return (
+        <section>
+            <Slider onChangeCommitted={() => {}} min={0} max={100} value={30} />
+        </section>
+    );
+}
+```
+
+Можно указать два значения.
```
Comment (Yeti-or) [inline website/plasma-web-docs/docs/components/Slider.mdx:31]: может так?
```
можно использовать диапазон значений
```

**Review by Yeti-or — APPROVED** [review]

**Comment by Yeti-or** [comment]
Screenshot 2023-03-06 at 12 23 22


что-то поломалось в доке https://plasma.sberdevices.ru/pr/pr-379/web/components/Upload/

**Comment by Yeti-or** [comment]
Editable тоже плохо себя ведет – скачет в интерфейсе =/

https://user-images.githubusercontent.com/1813468/223070204-6294d6a4-7b65-49f2-aded-f6acd3750f57.mov

**Comment by Yakutoc** [comment]
@Yeti-or В доке у нас есть такие вот правила

```css
... > div {
    display: flex;
    gap: 1rem;
}
```

Что ломает иной раз поведение. 

Я не искал откуда и почему у нас такое берется и поэтому добавляю обертку из `<section>...</section>`.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-379/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-379/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-379/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-379/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-379/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-379/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-379/temple-storybook/

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**